lib/fog/vcloud_director/models/compute/vapp.rb

```diff
@@ -16,19 +16,19 @@ class Vapp < Model
         attribute :owner
         attribute :maintenance, :type => :boolean
 
-        def initialize(*args, **kwargs)
+        def initialize(attributes = {})
```

Comment:
The [base class](https://github.com/xlab-si/fog-vcloud-director/blob/master/lib/fog/vcloud_director/compute.rb#L271) is just one hash argument

Reply:
Wow, good find.